src/core/server/http/http_server.ts

```diff
@@ -122,6 +123,12 @@ export class HttpServer {
         isAuthenticated: this.authState.isAuthenticated,
       },
       getAuthHeaders: this.authRequestHeaders.get,
+      getServerInfo: () => ({
+        name: config.name,
+        host: config.host,
+        port: config.port,
+        protocol: this.server!.info.protocol,
+      }),
       isTlsEnabled: config.ssl.enabled,
```

Comment:
Now that we got this `getServerInfo` API, should I move the `isTlsEnabled` info in the `HttpServerInfo` type and remove the direct access from the setup contract?

Reply:
makes sense 👍

Reply:
Ok, I'll do it in a separate PR later this week